requirements/pip.txt

```diff
@@ -1,10 +1,10 @@
 # Base packages
-pip==9.0.3
+pip==10.0.0
 appdirs==1.4.3
-virtualenv==15.1.0
+virtualenv==15.2.0
 docutils==0.14
-Sphinx==1.7.0
-sphinx_rtd_theme==0.2.5b1
+Sphinx==1.7.2
+sphinx_rtd_theme==0.3.0
```

Comment:
I think we should wait until https://github.com/rtfd/sphinx_rtd_theme/issues/610 is solved, right?

Reply:
Good call. Let's hold off on this one for now. I'm going to try to get the issues around this resolved.

Reply:
I will stick in `0.2.5b1` until that issue is solved.